lsp-mode.el

```diff
@@ -1197,17 +1197,15 @@ the lists according to METHOD."
                                      (-compose #'not #'lsp-empty?)
                                      results)
                                (`(,hover) hover)
-                               (hovers (-reduce-from
-                                        (-lambda ((&Hover :contents) result)
-                                          (lsp:set-hover-contents
-                                           result (append
-                                                   (if (and (sequencep contents)
-                                                            (not (stringp contents)))
-                                                       contents
-                                                     (list contents))
-                                                   (lsp:hover-contents result))))
-                                        (lsp-make-hover)
-                                        hovers))))
+                               (hovers (lsp-make-hover
+                                        :contents
+                                        (apply #'append (-map
+                                                         (-lambda ((&Hover :contents))
+                                                           (if (and (sequencep contents)
+                                                                    (not (stringp contents)))
+                                                               (append contents ())
+                                                             (list contents)))
+                                                         hovers))))))
```

Comment:
Wouldn't hovers always be nil here, causing hovers to always be empty in this code path? This also seems present in the old code, but I don't see it.

Reply:
Never mind, I forgot how `pcase` works for a second...